pkg/tracer/common/model.go

```diff
@@ -156,28 +156,33 @@ func (m *Histogram) MarshalJSON() ([]byte, error) {
 	return json.Marshal(encoded)
 }
 
-type ConnTupleV4 struct {
+type ConnTuple struct {
```

Comment:
Should we create another Tuple, like the C code:
```
struct ipv6_tuple_t {
	/* Using the type unsigned __int128 generates an error in the ebpf verifier */
	__u64 laddr_h;
	__u64 laddr_l;
	__u64 raddr_h;
	__u64 raddr_l;
	__u16 lport;
	__u16 rport;
	__u32 netns;
	__u32 pid;
};
```

Reply:
Not needed, the golang code is only interested in Laddr, Raddr, Lport, Rport, and Pid.